src/sunmemory/sycl/sundials_sycl_memory.cpp

```diff
@@ -89,31 +91,25 @@ SUNMemoryHelper SUNMemoryHelper_Sycl(SUNContext sunctx)
 
 SUNMemoryHelper SUNMemoryHelper_Clone_Sycl(SUNMemoryHelper helper)
```

Comment:
Add `SUNCheckLastErrNull` after `SUNMemoryHelper_NewEmpty` above and `SUNAsserNull` after `malloc` above